cms/src/types/dto.ts

```diff
@@ -77,3 +86,19 @@ export type ChangeReqAckDto = {
     message?: string;
     doc?: any;
 };
+
+export enum AclPermission {
+    View = "view",
+    Assign = "assign",
+    Edit = "edit",
+    Translate = "translate",
+    Publish = "publish",
+    Delete = "delete",
+}
+
+// export type AccessMap = Map<Uuid, Map<DocType, Map<AclPermission, boolean>>>;
+export type AccessMap = {
+    [a: Uuid]: { [b in DocType]?: { [c in AclPermission]?: boolean | undefined } };
```

Comment:
Nitpick: for types commonly `T` is used as a placeholder, followed by `U`, `V` etc.

Also, there's a commented line with an older version above this one.

Reply:
Fixed, thanks